### PR TITLE
Tool to merge to hex files into one large hex file

### PR DESCRIPTION
**Pull request by EricB-ADI**

### CODE REVIEW

**Review by yc-adi — APPROVED** [review]

File: Tools/misc/hexmerge.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+ #
+ # The mere transfer of this software does not imply any licenses
+ # of trade secrets, proprietary technology, copyrights, patents,
+ # trademarks, maskwork rights, or any other form of intellectual
+ # property whatsoever. Maxim Integrated Products, Inc. retains all
+ # ownership rights.
+ #
+ ###############################################################################
+
+## dtm_sweep.py
```
Comment (kevin-gillespie) [inline Tools/misc/hexmerge.py:36]: Update or remove these comments.

**Review by kevin-gillespie — CHANGES_REQUESTED** [review]

**Comment by kevin-gillespie** [comment]
What's the purpose of this tool?

**Comment by EricB-ADI** [comment]
We used it to merge the hex of the bootloader and the otas

**Comment by kevin-gillespie** [comment]
We were able to include the .bin file with the gcc tools, so I don't think we need this.

**Pull request closed without merging**